README.md

```diff
@@ -363,12 +158,12 @@ compile 'com.squareup.mortar:mortar:(latest version)'
 
 ## Full Disclosure
 
-This stuff is in rapid development, and has been open sourced at an earlier
-time in its life than is typical for Square. While we have a lot of code
-written in this style, we are still in the process of migrating to Mortar per
-se. As its use broadens bugs will be fixed and apis will be broken. (Ideally
-the apis will need to change because of new uses that you find!)
+This stuff has been in "rapid" development over a pretty long gestation period, 
+but is finally stabilizing. We don't expect drastic changes before cutting a
+1.0 release, but we still cannot promise a stable API from release to release.
 
+Every Android app at Square is written in this View-centric style, and most
+of them use Mortar itself, including our flagship [Register][register] app. 
```

Comment:
No need to justify the View-centric architecture.  If you're looking to make a Mortar testimonial, you can just simplify this to something like "Mortar is a key component of multiple Square apps, including our flagship, Square Register."